references.bib

```diff
@@ -678,4 +678,18 @@ @book{judson2020abstract
     title={Abstract Algebra: theory And Applications},
     author={Judson, Thomas W},
     year={2020}
+  }
+  @misc{directproduct,
+    title = {Direct product and Direct Sum},
+    author = {R.A Bailey},
+    publisher = {Queen Mary University of London},
+    pages = {4},
+    url = {https://webspace.maths.qmul.ac.uk/r.a.bailey/MAS305/algnotes14.pdf},
+    year = {2023}
+  }
+  @misc{szabo2000linear,
```

Comment:
Is this truly a 'misc' or is there a better way to refer to this work?

Reply:
It is a book. I fixed it.